tests/tensorlake/test_graph_api.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+from typing import List, Union
+
+import parameterized
+from testing import remote_or_local_graph, test_graph_name
+
+from tensorlake import Graph, RouteTo, tensorlake_function
+
+
+@tensorlake_function()
+def a(value: int) -> int:
+    return value + 3
+
+
+@tensorlake_function()
+def b(value: int) -> int:
+    return value + 4
+
+
+@tensorlake_function()
+def c(value: int) -> int:
+    return value + 5
+
+
+@tensorlake_function(next=[a, b])
+def fan_out(value: int) -> int:
+    return value
+
+
+@tensorlake_function(next=[a, b, c])
+def route_out(value: int) -> RouteTo[int, Union[a, b, c]]:
+    if value % 2 == 0:
+        return RouteTo(value + 1, [a])
+
+    return RouteTo(value + 2, [b, c])
```

Comment:
What about the syntax we discussed previously:
`RouteTo(b=value + 2, c=value +2)`
Is it excessive for now?

Reply:
Yeah -- I liked the idea, but it added more complication to the system; it's a bigger change than we really need right now.